LICENSE-binary

```diff
@@ -0,0 +1,531 @@
+This product bundles various third-party components under other open source
+licenses. This section summarizes those components and their licenses.
+See licenses-binary/ for text of these licenses.
+
+
+BSD 2-Clause
+------------
+
+hadoop-common-project/hadoop-common/src/main/native/src/org/apache/hadoop/io/compress/lz4/{lz4.h,lz4.c,lz4hc.h,lz4hc.c}
+hadoop-hdfs-project/hadoop-hdfs-native-client/src/main/native/fuse-dfs/util/tree.h
+hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-nodemanager/src/main/native/container-executor/impl/compat/{fstatat|openat|unlinkat}.h
+
+dnsjava:dnsjava:2.1.7
+org.codehaus.woodstox:stax2-api:3.1.4
+
+
+BSD 3-Clause
+------------
+
+hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/util/bloom/*
+hadoop-common-project/hadoop-common/src/main/native/gtest/gtest-all.cc
+hadoop-common-project/hadoop-common/src/main/native/gtest/include/gtest/gtest.h
+hadoop-common-project/hadoop-common/src/main/native/src/org/apache/hadoop/util/bulk_crc32_x86.c
+hadoop-hdfs-project/hadoop-hdfs-native-client/src/main/native/fuse-dfs/util/tree.h
+hadoop-tools/hadoop-sls/src/main/html/js/thirdparty/d3.v3.js
+hadoop-hdfs-project/hadoop-hdfs/src/main/webapps/static/d3-3.5.17.min.js
+leveldb
+
+com.google.protobuf:protobuf-java:2.5.0
+com.google.protobuf:protobuf-java:3.6.1
+com.google.re2j:re2j:1.1
+com.jcraft:jsch:0.1.54
+com.thoughtworks.paranamer:paranamer:2.3
+javax.activation:javax.activation-api:1.2.0
+org.fusesource.leveldbjni:leveldbjni-all:1.8
+org.jline:jline:3.9.0
+org.hamcrest:hamcrest-core:1.3
+org.ow2.asm:asm:5.0.4
+org.ow2.asm:asm-commons:6.0
+org.ow2.asm:asm-tree:6.0
+
+
+MIT License
+-----------
+
+hadoop-hdfs-project/hadoop-hdfs/src/main/webapps/static/dataTables.bootstrap.js
+hadoop-hdfs-project/hadoop-hdfs/src/main/webapps/static/dataTables.bootstrap.css
+hadoop-hdfs-project/hadoop-hdfs/src/main/webapps/static/jquery.dataTables.min.js
+hadoop-yarn-project/hadoop-yarn/hadoop-yarn-common/src/main/resources/webapps/static/dt-1.10.18/*
+hadoop-hdfs-project/hadoop-hdfs/src/main/webapps/static/dust-full-2.0.0.min.js
+hadoop-hdfs-project/hadoop-hdfs/src/main/webapps/static/dust-helpers-1.1.1.min.js
+hadoop-hdfs-project/hadoop-hdfs/src/main/webapps/static/moment.min.js
+hadoop-hdfs-project/hadoop-hdfs/src/main/webapps/static/angular-nvd3-1.0.9.min.js
+hadoop-hdfs-project/hadoop-hdfs/src/main/webapps/static/angular-1.6.4.min.js
+hadoop-hdfs-project/hadoop-hdfs/src/main/webapps/static/angular-route-1.6.4.min.js
+hadoop-hdfs-project/hadoop-hdfs/src/main/webapps/static/jquery-3.3.1.min.js
+hadoop-tools/hadoop-sls/src/main/html/js/thirdparty/jquery.js
+hadoop-yarn-project/hadoop-yarn/hadoop-yarn-common/src/main/resources/webapps/static/jquery
+hadoop-yarn-project/hadoop-yarn/hadoop-yarn-common/src/main/resources/webapps/static/jt/jquery.jstree.js
+hadoop-hdfs-project/hadoop-hdfs/src/main/webapps/static/bootstrap-3.3.7
```

Comment:
These lines are not covered in LICENSE.txt. These could be sorted with above lines.

Reply:
Sorted. Thanks.